src/domain/entities/ValidationRuleGroup.ts

```diff
@@ -0,0 +1,7 @@
+import { Id } from "./Ref";
+
+export type ValidationRuleGroup = {
```

Comment:
No need to change, but in a future you can use the NamedRef type for entities with Id, Name.

```ts
export type ValidationRuleGroup = NamedRef & {
    description: string;
};
```